### PR TITLE
Updates to testing

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

File: content/r-package-dev/Writing_Tests.Rmd [hunk 1]
```diff
@@ -41,9 +41,9 @@ Formal software testing lays down a safety net to ensure that software performs
...
 
-In developing a package, you will be adding code to perform certain functions.  In order to prove that this code does what it claims to, write a test that runs the function with different inputs and verify the output (or side effects).  Test edge cases and error conditions to make sure they are handled appropriately.  Creating tests covering expected behavior and edge cases as part of a testing framework in your package ensures the following:
+In developing a package, you will be adding code to perform certain functions.  In order to verify that this code does what it claims to, write a test that runs the function with different inputs and verify the output (or side effects).  Test edge cases and error conditions to make sure they are handled appropriately.  Tests are also meant to signal to future developers what the requirements of particular functionality are and to warn when those are not being met.  Creating tests covering expected behavior and edge cases as part of a testing framework in your package ensures the following:
```
Comment (wdwatkins) [inline content/r-package-dev/Writing_Tests.Rmd]: Do we need to define "edge cases" somewhere?

Comment (lindsaycarr) [inline content/r-package-dev/Writing_Tests.Rmd]: Hmmm that's an good point. @jiwalker-usgs what do you think?

Comment (jiwalker-usgs) [inline content/r-package-dev/Writing_Tests.Rmd]: I suppose it would be good to define that a bit better.

**Comment by lindsaycarr** [comment]
This addresses #155

**Comment by lindsaycarr** [comment]
Merging for now, we can talk about the need to define "edge cases" later on.
